### PR TITLE
magento/magento2-page-builder#543: Content Product Attributes Default…

### LINKED ISSUE #543: Content product attributes default sorting order
## Story
As a Marketer I want default content product attributes (short description and description) to be sorted in the content section of the default attribute set in the same order they appear on storefront *so that* it is aligned with storefront and I don't need to scroll through the description content (that can be long) to get access to short description (that's expected to be short)

## Business Value
better UX accessing short description field 
 
## Preconditions 
 * User is logged into Admin Panel
 * User is on Product Page with Default attribute set selected

## Acceptance Criteria
* User scrolls down to the content section and sees the sorting order of the fields is the following:
  ** short description
  ** description

#### Manual checks for ticket
- 

#### Automated MFTF Tests
- No new tests needed. No updated tests needed
-- Scenarios are not high enough priority 

#### Manual Hip Tests
- No new tests needed. No updated tests needed
-- Scenarios are not high enough priority

### PR DESCRIPTION
**Pull request by bluemwhitew**
### Description (*)
Reordered the `description` and `short_description` product attributes to better accommodate planned Page Builder content staging changes.

### Fixed Issues
1. Fixes magento/magento2-page-builder#543

### Builds
_N/A_

### Related Pull Requests
https://github.com/magento/magento2-page-builder/pull/585

### Questions or Comments
_N/A_

### Checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

---

#### Manual Testing Scenarios
- [ ] **New Magento Installation**
1. Create a new Product (**Catalog** » **Products** » **Add Product**)
2. Expand the **Content** accordion
3. Verify that **Short Description** appears _before_ **Description**
- [ ] **Existing Magento Installation**:
1. Create a new Product (**Catalog** » **Products** » **Add Product**)
2. Expand the **Content** accordion
3. Verify that **Description** appears _before_ **Short Description**
4. Update to include this changeset
5. Run `php bin/magento setup:upgrade`
6. Create a new Product (**Catalog** » **Products** » **Add Product**)
7. Expand the **Content** accordion
8. Verify that **Short Description** appears _before_ **Description**
- [ ] **Attribute Sorting in Admin**:
1. Modify Attribute Sets (**Stores** » **Attributes** » **Attribute Set**)
2. Click on the "Default" Attribute Set
3. Switch the order of `description` and `short_description` respectively
4. Click **Save**
5. Create a new Product (**Catalog** » **Products** » **Add Product**)
6. Expand the **Content** accordion
7. Verify that **Description** appears _before_ **Short Description**

#### Automated MFTF Tests
- No new tests needed
-- Scenarios are not high enough priority 
- Update existing tests to pass

#### Manual Hip Tests
- No new tests needed. No updated tests needed
-- Scenarios are not high enough priority

### CODE REVIEW

File: app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductAttributes.php [hunk 1]
```diff
@@ -113,14 +113,15 @@ public function apply()
...
+            100
         );
         $categorySetup->addAttributeToGroup(
             \Magento\Catalog\Model\Product::ENTITY,
             $attributeSetId,
             'Content',
-            'short_description',
-            100
+            'description',
+            110
```
Comment (omiroshnichenko) [inline app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductAttributes.php]: This patch will work only for installation. You need to write a new patch.

File: app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            UpdateMediaAttributesBackendTypes::class,
+        ];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getVersion()
+    {
+        return '2.3.5';
```
Comment (bluemwhitew) [inline app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php:97]: Please advise on this.

/cc @omiroshnichenko

File: app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    {
+        return [
+            UpdateMediaAttributesBackendTypes::class,
+        ];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getVersion()
```
Comment (omiroshnichenko) [inline app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php]: getVersion is redundant

**Comment by bluemwhitew** [comment]
@magento run all tests

**Comment by bluemwhitew** [comment]
@magento run all tests

**Comment by bluemwhitew** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Review by omiroshnichenko — CHANGES_REQUESTED** [review]
Please, fix failed tests

**Comment by bluemwhitew** [comment]
@magento run all tests

**Review by omiroshnichenko — DISMISSED** [review]

**Comment by bluemwhitew** [comment]
@magento run Static Tests

File: app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php [hunk 4]
```diff
@@ -0,0 +1,86 @@
+<?php
```
Comment (dhaecker) [inline app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php:1]: @omiroshnichenko @bluemwhitew do we need to add new integration tests for these new php functions?

**Comment by bluemwhitew** [comment]
@magento run all tests

File: app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php (hunk 4 above)
Comment (omiroshnichenko) [reply app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php:1]: no

**Review by omiroshnichenko — APPROVED** [review]

**Review by dhaecker — APPROVED** [review]

**Comment by bluemwhitew** [comment]
@magento run all tests

**Review by VladimirZaets — APPROVED** [review]

**Review by omiroshnichenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-7918](https://jira.corp.magento.com/browse/ENGCOM-7918) has been created to process this Pull Request
:eight_spoked_asterisk: @VladimirZaets, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by dhaecker** [comment]
@magento run Semantic Version Checker

**Pull request merged**